embedchain/embedchain.py

```diff
@@ -154,25 +157,35 @@ def _format_result(self, results):
             )
         ]
 
-    def get_openai_answer(self, prompt):
+    def get_llm_answer(self, prompt):
         messages = []
         messages.append({
             "role": "user", "content": prompt
         })
-        response = openai.ChatCompletion.create(
-            model="gpt-3.5-turbo-0613",
-            messages=messages,
-            temperature=0,
-            max_tokens=1000,
-            top_p=1,
-        )
+        if Local_llm_flag:  
+            response = HuggingFaceTextGenInference(  
+                inference_server_url=os.getenv('INFERENCE_SERVER_URL'),
+                max_new_tokens=1000,  
+                top_k=10,  
+                top_p=0.95,  
+                typical_p=0.95,  
+                temperature=0.01,  
+                repetition_penalty=1.03,  
+            )
```

Comment:
Is the prompt actually used in this inference gen?

I think it can just return: `return response(prompt)` here as the return type also seems to be different (str), but otherwise, this PR helped me a lot! :100:

Reply:
Well then should we also change the `openai.ChatCompletion` response? I added the parameters to rhe HF response to match what existed with the OpenAI response:

```python
response = openai.ChatCompletion.create(
                model="gpt-3.5-turbo-0613",
                messages=messages,
                temperature=0,
                max_tokens=1000,
                top_p=1,
            )
        return response["choices"][0]["message"]["content"]
```

Reply:
I believe you are correct, I didn't separate the `response` between the inference functions. I have updated the PR to reflect this oversight. Thank-you.